hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -208,6 +209,11 @@
       .sinceVersion("0.11.0")
       .withDocumentation("Table checksum is used to guard against partial writes in HDFS. It is added as the last entry in hoodie.properties and then used to validate while reading table config.");
 
+  public static final ConfigProperty<Boolean> METADATA_TABLE_ENABLE = ConfigProperty
```

Comment:
Are we sure there is no need to handle this config in upgrade step? Would it be better to add this config in hoodie.properties when users upgrade with an existing metadata table?

Reply:
Mentioned before. Thanks a lot for your review!